Add 4 builder statistics

EtienneMR opened the pull request:
# Changes proposed in this pull request:
Adding 4 builder statistics
- Build built
- Build upgraded
- Build repaired
- Build removed

Review please

EtienneMR committed:
Added 4 builder statistics

- Build built
- Build upgraded
- Build repaired
- Build removed

EtienneMR committed:
Merge pull request #1 from ldtteam/version/main

Updating to latest version

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ldtteam/minecolonies?pullRequest=9067) <br/>All committers have signed the CLA.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ldtteam/minecolonies?pullRequest=9067) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ldtteam/minecolonies?pullRequest=9067) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ldtteam/minecolonies?pullRequest=9067) it.</sub>

Thodor12 commented on src/main/resources/assets/minecolonies/lang/manual_en_us.json:
```diff
@@ -2371,6 +2371,10 @@
   "com.minecolonies.coremod.gui.townhall.stats.crops_harvested": "Crops Harvested: %d",
   "com.minecolonies.coremod.gui.townhall.stats.land_tilled": "Land Tilled: %d",
   "com.minecolonies.coremod.gui.townhall.stats.fish_caught": "Fish Caught: %d",
+  "com.minecolonies.coremod.gui.townhall.stats.build_built": "Build Built: %d",
```
Would make this "Buildings ..."

EtienneMR replied:
I changed it

Raycoms reviewed (changes requested):
Good job on this! This is looking good already. I agree with thodor. I would word it "Building ...." too

EtienneMR committed:
Updated builder statistics names

Changed
Build built to Building built
Build upgraded to Building upgraded
Build repaired to Building repaired
Build removed to Building removed
